src/main/kotlin/no/digdir/fdk/searchservice/service/SearchService.kt

```diff
@@ -111,7 +111,21 @@ class SearchService(
             })
         }
 
-        return queryFilters
+        filters?.spatial?.let { spatial ->
+            val filterList = spatial.split(",").map { it.trim() }
+
+            filterList.forEach { spatialValue ->
+                queryFilters.add(DSLQuery.of { queryBuilder ->
+                    queryBuilder.term { termBuilder ->
+                        termBuilder
+                            .field("spatial.code.keyword")
```

Comment:
Er ikke `code` som blir brukt av fulltext for dette filteret, ref: https://github.com/Informasjonsforvaltning/fdk-fulltext-search/blob/main/src/fdk_fulltext_search/search/query_filter_utils.py#L19C17-L19C45

Men det er kanskje det vi burde gjøre, det vil dog kreve enda mer refaktorering av fdk-portal...

Reply:
Byttet til å bruke prefLabel